doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -81,33 +76,37 @@ enable more features, such as distributed memory execution.
 
 .. _using-a-more-full-featured-chapel:
 
-Using a More Full-Featured Chapel
----------------------------------
+Using Chapel in its Preferred Configuration
+-------------------------------------------
 
-To use Chapel in a more full-featured and preferred configuration,
-you will need to rebuild Chapel from source in a different configuration.
+To use Chapel in its preferred, full-featured mode, you will need to
+rebuild Chapel from source in a different configuration:
 
 *  Open up a new shell to avoid inheriting the previous environment
    settings.
 
-*  The quickstart configuration described above sets ``CHPL_LLVM=none``
-   but this is not the preferred configuration of Chapel. As of 1.25,
-   LLVM is the default compiler backend and it needs to be available
-   for full functionality. So make sure you have a compatible system LLVM
-   installed in a standard location or set the environment variable
-   ``CHPL_LLVM`` to ``bundled`` or ``none`` as described in
-   :ref:`chplenv.rst <readme-chplenv.CHPL_LLVM>`.  If desired, you must
-   explicitly opt-out of using it by setting
-   ``CHPL_LLVM=none``. The ``CHPL_LLVM=bundled`` build takes a long
-   time, so to avoid surprises you need to explicitly opt-in to using it.
-
-*  Repeat steps 2-5 above, but in Step 2, source ``util/setchplenv.bash``
+* The Quickstart configuration described above sets ``CHPL_LLVM=none``
+  for simplicity and to save time.  This causes the Chapel compiler to
+  use its C back-end, which is not the preferred option.  As of Chapel
+  1.25, LLVM is the default back-end, which needs to be available for
+  full functionality.  There are a few ways to use LLVM:
+
+  - ensure that you have a version of LLVM 11 installed on your system
+    and set ``CHPL_LLVM=system`` (or leave it unset and Chapel should
+    find it if it's in your path)
+
+  - set ``CHPL_LLVM=bundled`` to have Chapel build and use the bundled
+    version of LLVM (note that this can take a *long* time)
```

Comment:
I used:

>  (note that building the bundled version of LLVM can take a *long* time)

because it's not really the building of the Chapel compiler itself that takes a particularly long time, but the LLVM stage itself.